packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -123,13 +192,24 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
             locationPathname,
             viewConfig: {
                 direction: Direction.ltr,
+                theme: ThemeName.light,
+                backgroundColor: DesignSystemDefaults.backgroundColor,
+                designSystem: Object.assign({}, DesignSystemDefaults, {
+                    neutralPalette:
+                        neutralColorPalette !== null
+                            ? createColorPalette(neutralColorPalette)
+                            : [],
+                    direction: Direction.ltr,
+                }),
             },
         };
     }
 
     public render(): React.ReactNode {
         return (
-            <DesignSystemProvider designSystem={DesignSystemDefaults}>
+            <DesignSystemProvider
```

Comment:
This will create a new object every render cycle and cause components to recalculate styles every render cycle. You should store this as a variable and assign the variable to the `designSystem` prop